app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/EateryCard.kt

```diff
@@ -43,25 +46,31 @@ import androidx.compose.ui.graphics.ImageBitmap
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
-import androidx.lifecycle.compose.ExperimentalLifecycleComposeApi
 import com.cornellappdev.android.eatery.R
 import com.cornellappdev.android.eatery.data.models.Eatery
 import com.cornellappdev.android.eatery.data.repositories.CoilRepository
+import com.cornellappdev.android.eatery.ui.components.details.PaymentWidgets
```

Comment:
There are lots of unused imports in this file, make sure you go to settings -> Actions on save -> optimize imports so these don't get through